DisscResultsOYAFR.Rmd

```diff
@@ -0,0 +1,44 @@
+---
+title: "R Notebook"
+output:
+  html_document: default
+  html_notebook: default
+---
+
+```{r}
+library(tidyverse)
+library(ggplot2)
+library(lme4)
+library(broom)
+library(ggthemes)
+library(MuMIn)
+
+devtools::load_all(".")
+```
+Results:
+
+*Offspring per Year, Age of First Repoduction Model.*
+
+The number of offspring per year was modeled against the average age of first reproduction. The model created was a general linear model that used the data points for each species. This model showed that there is a negative relationship between the age when a mammal species will first reproduce and the number of offspring produced per year. See the Firgure 2 for plotted line, and the slope of the line is -0.89404.The second model using these two variables added in a randomization effect that grouped the data points by order. This model also found that there is a negative relationship between age when a mammal species will first reproduce and the number of offspring produced per year. This trend holds true for all of the orders, as their slopes are all negative, as can be seen in Figure 2. Slopes of the lines for each of the orders are located in Table 2. These two models were then compared to each other to see what one fits the data better. From this analysis, the better model is the one that accounts for the orders. See table 1 for AIC values. 
```

Comment:
Since "better" can be subjective depending on your goals, try to describe why one model is preferable. For example, you don't always want to maximize how the model fits your current data, as this can leads to overfitting and poor generalization to new observations, so this metric does not necessarily indicate the "better" model (but you could rephrase to explain why this is the most suitable metric and model for your study).